src/main/java/InputParser.java

```diff
@@ -0,0 +1,66 @@
+public class InputParser {
+    private final String[] words;
+
+    public InputParser(String sentence) {
+        this.words = sentence.split(" ");
+    }
+
+    /**
+     * Gets the command from input regardless if the command is valid or not.
+     * The command should be the first word of the input.
+     *
+     * @return the command for the input
+     */
+    public String getCommand() {
+        return words[0].toLowerCase();
+    }
+
+    /**
+     * Gets the description of the task from input which excludes the command
+     * and date if there is any.
+     *
+     * @return the description of the task
+     */
+    public String getDescription() {
+        if (words.length <= 1) {
+            return "";
+        }
+        String description = words[1];
+        for (int i = 2; i < words.length; i++) {
+            if (words[i].startsWith("/")) {
+                break;
+            }
+            description += " " + words[i];
+        }
```

Comment:
Good job making the happy path prominent. In other words, it's good that you made the execution path when everything goes well, distinct from the unusual case.